### DISCUSSION TITLE
Problem: use_isomorphic_layout_effect_cjs is not a function

### DISCUSSION THREAD

**Discussion by bbg** [TanStack/form#468] [Q&A] [answered]
Greetings,

I used Tanstack Form in my project with NextJS/React but I get this error "TypeError: use_isomorphic_layout_effect_cjs is not a function" during build not during development. Can you help me how to solve this problem?

![CleanShot 2023-09-17 at 14 13 02@2x](https://github.com/TanStack/form/assets/13348876/9fe403cf-1b7a-4c5c-bbd1-1fd2061bfcf3)

**Comment by crutchcorn** [comment]
This must be a bug in TSUp somehow.

Can you open a GitHub issue with a minimal reproduction?

**Comment by crutchcorn** [comment, marked as answer]
This is a fixed bug
